R/workflows-package.R

```diff
@@ -12,5 +12,7 @@
 #' @importFrom parsnip fit_xy
 #'
 #' @importFrom stats predict
+#'
+#' @importFrom utils capture.output
```

Comment:
i.e. remove this too and redocument. When you merged in master i think you got the fix